rest/server_context.go

```diff
@@ -1120,6 +1120,9 @@ func (sc *ServerContext) initEventHandlers(ctx context.Context, dbcontext *db.Da
 				}
 			}
 			conf.Filter = filter
+			if conf.Filter == "" {
+				base.WarnfCtx(ctx, "No filter specified in webhook configuration or filter has failed to be retrieved")
```

Comment:
I think we should log this at info level (with the `Events` log key), as it can be expected that a remote URL sets no filter in a 200 response, and is not something that needs to be actionable by a customer.

The error/warning cases are when the remote URL returns a non-200 status code, which we already should have error handling for.

```suggestion
				base.InfofCtx(ctx, base.KeyEvents, "No filter function defined for event handler %s - everything will be processed", eventType.String())
```